node/resolvers/checkout/index.ts

```diff
@@ -54,8 +72,30 @@ export const fieldResolvers = {
 }
 
 export const queries: Record<string, Resolver> = {
-  orderForm: (root, args, {dataSources: {checkout}}) => {
-    return checkout.orderForm()
+  orderForm: async (root, args, ctx) => {
+    const {dataSources: {checkout, session}, request: { headers: { cookie } }} = ctx
+
+    const cookieParsed = parse(cookie)
+    const coOrderFormId = cookieParsed['checkout.vtex.com'] && cookieParsed['checkout.vtex.com'].split('=')[1]
+    const sessionData = await sessionQueries.getSession(root, args, ctx) as SessionFields
+    if (sessionData.orderFormId) {
+      if (!coOrderFormId) {
+        ctx.request.headers.cookie = `${cookie}; checkout.vtex.com=__ofid=${sessionData.orderFormId}`
+      } else if (sessionData.orderFormId !== coOrderFormId) {
```

Comment:
Saving the orderFormId should happen always, right? Not only if it's already in the session.

Reply:
If the OFID in the session is === to the one in the cookie, why waist time saving it?